src/components/TransactionForm.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import FormButtons from './FormButtons';
+import 'antd/dist/antd.css';
+import { Input, message } from 'antd';
+import { ShopOutlined, TableOutlined, DollarOutlined } from '@ant-design/icons';
+
+class TransactionForm extends Component {
+    constructor() {
+        super()
+        this.state = {
+            amount: '',
+            vendor: '',
+            category: ''
+        }
+    }
+
+    handleInputChange = (e) => {
+        const target = e.target;
+        const value = target.value;
+        const name = target.name;
+
+        this.setState({ [name]: value });
+    }
+
+    invalidInputs(amount, vendor, category) {
+        if (!amount || !vendor || !category || amount === 0) {
+            return true;
+        }
+        return false;
```

Comment:
you can just return the expression `return (!amount || !vendor || !category || amount === 0)`